controllers/comments.controllers.js

```diff
@@ -0,0 +1,8 @@
+const { deleteComment } = require("../models/comments.models")
+
+exports.deleteCommentById = (req, res, next) => {
+    const {comment_id} = req.params
+    deleteComment(comment_id)
+    .then(() => res.status(204).send())
```

Comment:
could also use res.sendStatus(204) here which skips the need for .send() when you're not passing back a body 🧠